UI/CMakeLists.txt

```diff
@@ -84,6 +84,23 @@ target_sources(
           ui-validation.cpp
           ui-validation.hpp)
 
+target_sources(
+  obs-studio
+  PRIVATE # cmake-format: sortable
+          goliveapi-censoredjson.cpp
+          goliveapi-censoredjson.hpp
+          goliveapi-network.cpp
+          goliveapi-network.hpp
+          goliveapi-postdata.cpp
+          goliveapi-postdata.hpp
```

Comment:
"goliveapi" is clearly Twitch/Amazon made thing. I would really prefer that most of this goes to where it belong: in the service plugin side.

We already have the Broadcast feature from YT to untangle to a proper service with an agnostic API to manage broadcast, this would worsen the situation.